lesson4/lib/news/services/Newsru.js

```diff
@@ -0,0 +1,20 @@
+const News = require('./News')
+
+class Newsru extends News {
+  constructor() {
+    super()
+    this.newsServiceUrl = 'http://www.newsru.com'
+    this.newsListUrn = '/allnews'
+    this.listItemSelector = '.index-news-content'
+    this.itemTitleSelector = '.article-title'
+    this.itemDateSelector = '.article-date'
+    this.itemBodySelector = '.article-text p.maintext'
+  }
+
+  getNewsItemDate($) {
+    let date = $(this.itemDateSelector).text().trim()
+    return date.substring(date.indexOf(':') + 2, date.indexOf('|')).trim()
```

Comment:
You should take a look at your previous PR. I have noticed the same

Reply:
Yes. I just did.